packages/next/server/web/sandbox/context.ts

```diff
@@ -208,6 +212,26 @@ Learn More: https://nextjs.org/docs/messages/middleware-dynamic-wasm-compilation
           init.headers.set(`user-agent`, `Next.js Middleware`)
         }
 
+        const inputString = String(input)
+        if (inputString.startsWith('/_next/static/media/')) {
+          const pathname = inputString.replace('/_next/static/media/', '')
+          const fullPathname = path.join(
+            options.distDir,
+            `server/static/media/${pathname}`
+          )
+          if (
+            await fs.access(fullPathname).then(
+              () => true,
+              () => false
+            )
+          ) {
+            const blob = createReadStream(fullPathname)
+            return new context.Response(requestToBodyStream(blob))
+          }
```

Comment:
Shall we throw something meaningful when it can't access to the file (and when will that happen?)

Reply:
Should we throw? or just "continue" and try to fetch?

Reply:
ah we already check for access and "continue" the routing chain if you don't have access